addons/web/static/tests/legacy/mobile/views/fields/statusbar_field_tests.js

```diff
@@ -81,25 +81,11 @@ QUnit.module("Mobile Fields", ({ beforeEach }) => {
         });
 
         assert.doesNotHaveClass(
-            fixture.querySelector(".o_field_statusbar"),
+            fixture.querySelector(".o_statusbar_status"),
             "o_field_empty",
             "statusbar widget should have class o_field_empty in edit"
         );
-        assert.containsOnce(fixture, ".o_statusbar_status button.dropdown-toggle:visible");
-        assert.strictEqual(
-            $(".o_statusbar_status button.dropdown-toggle:visible").text().trim(),
-            "..."
-        );
```

Comment:
Keep these test cases, and you can update like this
`.o_statusbar_status button.dropdown-toggle:visible:disabled`